content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -917,3 +990,5 @@ object, but maintains complete control over what Pods are created and how work i
   the UNIX tool `cron`.
 * Practice how to configure handling of retriable and non-retriable pod failures
   using `podFailurePolicy`, based on the step-by-step [examples](/docs/tasks/job/pod-failure-policy/).
+* Practice how to configure your Jobs to handle pod failures independently per
```

Comment:
```suggestion
* Learn how to configure your Jobs to handle pod failures independently per
```

Reply:
I actually delete this point, as this task is no longer added. We decided to leave the content for the task for later, when combined with other features around failure handling.